website/src/components/pricing/PricingPlan.tsx

```diff
@@ -112,6 +113,10 @@ const FEATURE_INFO: Record<keyof Features, FeatureInfo> = {
         description:
             'Free and Team tiers limit the total amount of searchable code. Enterprise offers options that scale to any size codebase.',
     },
+    managedInstance: {
+        label: 'Managed instance (available add-on)',
```

Comment:
@jeanduplessis - is it possible to make this all on one line? "Add-on" is on its own line.